src/MizarModels/include/MizarModels/SamplingWithFrameTrackReportModel.h

```diff
@@ -172,26 +169,24 @@ class SamplingWithFrameTrackReportModelTmpl : public QAbstractTableModel {
                         title.toStdString(), time, *function_name));
   }
 
-  [[nodiscard]] static QString MakeTooltipForSlowdownColumn(
-      const std::string* function_name, double slowdown_percent,
-      const Baseline<QString>& baseline_title, const Comparison<QString>& comparison_title) {
-    return QString::fromStdString(
-        absl::StrFormat("The function \"%s\" is  %.3f%%\n"
-                        "slower in %s capture that it was in %s capture.\n"
-                        "Negative percentage represent a speed-up.",
-                        *function_name, slowdown_percent, comparison_title->toStdString(),
-                        baseline_title->toStdString()));
+  [[nodiscard]] static QString MakeTooltipForSlowdownColumn(const std::string* function_name,
+                                                            double slowdown_percent) {
+    return QString::fromStdString(absl::StrFormat(
+        "The function \"%s\" is  %.3f%%\n"
+        "slower in %s capture that it was in %s capture.\n"
+        "Negative percentage represent a speed-up.",
+        *function_name, slowdown_percent, orbit_mizar_base::kBaselineTitle->toStdString(),
+        orbit_mizar_base::kComparisonTitle->toStdString()));
```

Comment:
Nit: YOu could consider using QString::arg instead of absl::StrFormat. This would avoid conversions to std::string. (If you prefer the current way, this is also fine.)

Reply:
Yeah, good point. But wouldn't this reduce consistency, as the rest of the class relies on `absl::StrFormat`? 

Anyway, this is orthogonal to the PR.

Reply:
```
But wouldn't this reduce consistency, as the rest of the class relies on absl::StrFormat
```

Yes, in a way. That's why I left the choice to you.

Nevertheless there are situations where QString can't be avoided because it has fully fledged encoding (UTF-8) support while std::string is merely a bunch of bytes. absl::StrFormat also can't handle anything that goes beyond ASCII.

TLDR: If we wanted to support other languages in the future, then we needed to move all formatting to QString::arg anyway. Hence there will always be use cases for both in the code base.

But you're right. It's orthogonal.